info-commands.go

```diff
@@ -341,8 +341,8 @@ type ServerProperties struct {
 // the number of calls of each API and the moving average of
 // the duration, in nanosecond, of each API.
 type DiskMetrics struct {
-	APILatencies map[string]interface{} `json:"apiLatencies,omitempty"`
-	APICalls     map[string]uint64      `json:"apiCalls,omitempty"`
+	LastMinute map[string]TimedAction `json:"last_minute,omitempty"`
+	APICalls   map[string]uint64      `json:"apiCalls,omitempty"`
 }
```

Comment:
There are some breaking changes here @klauspost - it is going to be tricky to perhaps managing the client versions and console UI versions.

Reply:
If we do this with semantic versioning we should move madmin-go to v2.0.0

Reply:
@harshavardhana There is no breakage on Console. It does not use `APILatencies`.

For `mc` the changes are compatible:

Old mc, New MinIO -> No `APILatencies` are shown in `mc admin info --json`, but everything otherwise works. Solution is to upgrade `mc`.

New mc, Old MinIO -> No `LastMinute ` are shown in `mc admin info --json`, but everything otherwise works. Solution is to downgrade `mc` (keep it at server level) or upgrade MinIO server.

So nothing stops working.